python_ta/reporters/persistent_server.py

```diff
@@ -0,0 +1,72 @@
+"""This module defines a HTML server using aiohttp that supports reloading via WebSockets."""
+
+import asyncio
+import logging
+import threading
+import webbrowser
+
+from aiohttp import WSMsgType, web
+
+logging.getLogger("aiohttp.access").disabled = True
+latest_html: bytes = b"<h1>Loading report...</h1>"
```

Comment:
Overall, we should not use global variable. In this case, you can encapsulate everything in this file into a class, and make all of these instance attributes.

Separately, it's reasonable to have a constant that defines the default HTML here, perhaps called `LOADING_HTML` or something similar.